copy_number/facetsFillGeneCN.R

```diff
@@ -154,8 +168,8 @@ OverCall <- function(column.name) {
             mutate(index=as.numeric(index)) %>%
             group_by(index) %>%
```

Comment:
I'm grouping by the unique row index, left joining to each group, and then selecting the closest row from the second df with slice().